Tests/RoutingTests/RoutableTests.swift

```diff
@@ -80,6 +80,18 @@ final class RoutableTests: XCTestCase {
         XCTAssertEqual(router.stack.count, 2)
         XCTAssertEqual(router.stack, [.settings, .profile])
     }
+
+    func testReplaceWithEmptyDestinations() {
+        router.replace(with: [])
```

Comment:
Would it make sense to push screens onto the stack before replacing?

Reply:
You're right. Will fix.